recipes/ocr_pre_annotations.ipynb

```diff
@@ -1,12 +1,23 @@
 {
```

Comment:
<p>We can now create the project ontology.</p><p> </p><p>For a transcription task on images, the ontology is a classification job with nested transcriptions for each category:</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1069/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>